src/Components/Members/MembersCreateEdit.js

```diff
@@ -0,0 +1,316 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router';
+import '../FormStyles.css';
+import { CKEditor } from '@ckeditor/ckeditor5-react';
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import TextField from '@mui/material/TextField';
+import Card from '@mui/material/Card';
+import Button from '@mui/material/Button';
+import CardHeader from '@mui/material/CardHeader';
+import { Alert } from '@mui/material';
+import { useDropzone } from 'react-dropzone';
+
+const thumb = {
+  display: 'inline-flex',
+  borderRadius: 2,
+  border: '1px solid #eaeaea',
+  margin: 'auto',
+  width: 'auto',
+  height: 60,
+  padding: 4,
+  boxSizing: 'border-box',
+};
+
+function MembersCreateEdit() {
+  const { id } = useParams();
+  const urlEdit = `http://ongapi.alkemy.org/public/api/members/${id}`;
+  const urlCreate = `http://ongapi.alkemy.org/public/api/members`;
+  const [initialValues, setInitialValues] = useState({
```

Comment:
please use a mor specif name, initialValues is too generic and do not give information about its content. Maybe member is a more specific name